<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_screen</class>
 <widget class="QMainWindow" name="game_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FLASH CARDS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(184, 217, 138);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="true_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-color:rgb(5, 119, 161);
background-color:rgb(135, 185, 225);
border-style: solid;
border-width: 1px;
border-radius: 25px;
color: rgb(170, 0, 0);
}
QPushButton::hover{
background-color: rgb(5, 119, 161);
color:white;
}
QPushButton::pressed{
padding-left:5px;
padding-top:5px;
	
	background-color: rgb(36, 83, 109);
}
</string>
    </property>
    <property name="text">
     <string>GOT IT ✔️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="false_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-color:rgb(5, 119, 161);
background-color:rgb(135, 185, 225);
border-style: solid;
border-width: 1px;
 border-radius: 25px;
color: rgb(170, 0, 0);
}
QPushButton::hover{
background-color: rgb(5, 119, 161);
color:white;
}
QPushButton::pressed{
padding-left:5px;
padding-top:5px;
background-color: rgb(36, 83, 109);
}

</string>
    </property>
    <property name="text">
     <string>STUDY AGAIN ✖️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>517</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-color: rgb(5, 119, 161);
background-color:rgb(135, 185, 225);
border-style: solid;
border-width: 1px;
border-radius:15px;
color: rgb(170, 0, 0);
}
QPushButton::hover{
background-color:rgb(5, 119, 161);
color:white;
}
QPushButton::pressed{
padding-left:5px;
padding-top:5px;
	
background-color: rgb(36, 83, 109);
}


</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>517</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-color: rgb(5, 119, 161);
background-color:rgb(135, 185, 225);
border-style: solid;
border-width: 1px;
border-radius:15px;
color: rgb(170, 0, 0);
}
QPushButton::hover{
background-color: rgb(5, 119, 161);
color:white;
}
QPushButton::pressed{
padding-left:5px;
padding-top:5px;
background-color: rgb(36, 83, 109);
}
</string>
    </property>
    <property name="text">
     <string>EXİT</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="true_score">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(170, 85, 0);

border-style: solid;
border-width: 1px;
border-radius:20px;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cards">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>450</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(170, 85, 0);
background-color:rgb(5, 119, 161);
border-style: solid;
border-width: 3px;
border-radius:50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="words_level">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>35</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
background-color: rgb(255, 255, 255);
color: rgb(170, 0, 0);
border-style:none;
border-radius:15px;
text-align:center;
}
QProgressBar::chunk{
border-radius:10px;
background-color: qlineargradient(x1: 1, y1: 0, x2: 1, y2: 0.5, stop: 0  rgb(205, 133, 63), stop: 1 rgb(255, 127, 63));
margin:1px;
height:20px;
}
</string>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="languages">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(205, 133, 63);</string>
    </property>
    <property name="text">
     <string>Language</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="words">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>451</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(205, 133, 63);</string>
    </property>
    <property name="text">
     <string>words</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="level_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(170, 85, 0);
background-color:rgb(135, 185, 225);
border-style: solid;
border-width: 1px;
border-radius: 15px;
color: rgb(170, 0, 0);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(170, 0, 0);


</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total_try">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(170, 85, 0);

border-style: solid;
border-width: 1px;
border-radius:20px;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timer_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(5, 119, 161);
background-color:rgb(135, 185, 225);
border-style: solid;
border-width: 1px;
 border-radius: 50px;
color: rgb(170, 0, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
